eng/pipelines/aggregate-reports.yml

```diff
@@ -128,3 +94,38 @@ jobs:
       condition: succeededOrFailed()
       displayName: 'Publish Report Artifacts'
       artifact: reports
+  - job: Compliance
+    timeoutInMinutes: 120
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'windows-2019'
+    steps:
+    - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
Component governance should remain in the other job as it requires the repository to be built and installed

Reply:
Does it? AFAIK, It scans the maven pom.xml files